### PR TITLE
test: Remove 3rd party api from cypress tests

### PR DESCRIPTION
**Pull request by Parthvi12**
## Description

This PR updates api used in test from 3rd party to TED's and fixes Omnibar spec

#### PR fixes following issue(s)

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Chore (housekeeping or task changes that don't impact user perception)
- This change requires a documentation update
>
>
>
- [ ] Manual
- [ ] Jest
- [ ] Cypress
>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

**Review by yatinappsmith — APPROVED** [review]

File: app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts [hunk 1]
```diff
@@ -108,7 +99,7 @@ describe("UUID Datatype tests", function () {
 
     agHelper.ClickButton("Generate UUID's");
     agHelper.AssertContains("All UUIDs generated & available");
-
+    cy.pause();
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts:102]: Is this needed for commit?

File: app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts [hunk 2]
```diff
@@ -313,28 +289,21 @@ describe("UUID Datatype tests", function () {
     });
     deployMode.DeployApp();
     table.WaitUntilTableLoad();
-    table.ReadTableRowColumnData(1, 5).then(($newFormedguid1) => {
-      expect($newFormedguid1).not.to.be.empty; //making sure new guid is set for row
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts:317]: Why is this removed?

File: app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts [hunk 3]
```diff
@@ -313,28 +289,21 @@ describe("UUID Datatype tests", function () {
...
-      dataSources.EnterQuery(query);
-      dataSources.RunQueryNVerifyResponseViews(1);
-      dataSources.AssertQueryResponseHeaders(["affectedRows"]);
-      dataSources.ReadQueryTableResponse(0).then(($cellData) => {
-        expect($cellData).to.eq("0");
-      });
-      deployMode.DeployApp();
-      table.WaitUntilTableLoad();
-      table.ReadTableRowColumnData(1, 5, "v1", 200).then(($newFormedguid2) => {
-        expect($newFormedguid1).to.eq($newFormedguid2);
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts:335]: Why is this removed?

File: app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts (hunk 2 above)
Comment (Parthvi12) [reply app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts:317]: @Aishwarya-U-R  We discussed to remove the guid api as its same as uuid, hence removed the table entry too

File: app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts (hunk 3 above)
Comment (Parthvi12) [reply app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts:335]: Same as above

File: app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts (hunk 2 above)
Comment (Aishwarya-U-R) [reply app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts:317]: No this is not guid. This is generating new UUID with a DB package called uuid_generate_v4() (refer line 307) and here it is checking if that value is generated & present.

File: app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts (hunk 3 above)
Comment (Aishwarya-U-R) [reply app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts:335]: Same as above, here it is generating via package gen_random_uuid() (refer line 325) and checking if its generated & present.

File: app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts (hunk 2 above)
Comment (Parthvi12) [reply app/client/cypress/e2e/Regression/ServerSide/Postgres_DataTypes/UUID_Spec.ts:317]: Okay, will add back in different PR then.
